### DISCUSSION TITLE
Is there a tutorial for taking a source voice audio, and a target voice audio, and making the target voice sound like the source voice as a new audio?

### DISCUSSION THREAD

**Discussion by TheInvoker** [serp-ai/bark-with-voice-clone#53] [General]
Is there a tutorial for taking a source voice audio, and a target voice audio, and making the target voice sound like the source voice as a new audio? The target audio will be singing, but the new saved audio should preserve the singing and only change the voice.

**Comment by devinschumacher** [comment]
> Is there a tutorial for taking a source voice audio, and a target voice audio, and making the target voice sound like the source voice as a new audio? The target audio will be singing, but the new saved audio should preserve the singing and only change the voice.

We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik
